lms/static/js/course_sharing/course_sharing_events.js

```diff
@@ -0,0 +1,29 @@
+/**
+* Module for emitting Course Sharing Events.
+*/
+(function(define) {
+    'use strict';
+
+    define(['jquery', 'logger'], function($, Logger) {
+        return function(courseId) {
+
+            $(".action-facebook[data-course-id='" + courseId + "']").on('click', function(){
+                // Emit an event telling that the Facebook share link was clicked.
+                Logger.log('edx.course.sharing.share_clicked', {
```

Comment:
@mulby For testing you can use this sandbox https://course-sharing-on-lms-dashboard.sandbox.edx.org/login?next=/dashboard